Paco-Server/codepush.properties

```diff
@@ -0,0 +1,19 @@
+#This property file will be used by the pre commit git hook
+#This will change only the values needed to be changed in the config xmls across all modules
+app.name=quantifiedself
+app.version=211
```

Comment:
We need to manually change the version in the properties file, in each environment (local/staging/staging2/codepush). 211, is just an example, I used.